scripts/gobuild.sh

```diff
@@ -48,4 +48,5 @@ else
 		   -ldflags "-s ${VERSION_FLAG} ${GIT_HASH_FLAG} ${GIT_DIRTY_FLAG}" \
 		   -o "${TOPWD}/amazon-ecs-init"
 fi
+go build -o "${TOPWD}/amazon-ecs-volume-plugin" "./volumes/amazon-ecs-volume-plugin"
```

Comment:
I see `amazon-ecs-init` build command above this has a few extra arguments as well as a dev build mode. I don't have enough context on what are the intentions, can you double check to see if we need to do something similar?

Reply:
Checked with @sharanyad and confirmed that we do not require those arguments or dev mode for this plugin.

Reply:
when packaging and signing RPMs, we will need the additional ceremony unfortunately. you can put a TODO here for now as long as we track this, though.

Reply:
Added a TODO, will change it after testing.